modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/Master/index.md

```diff
@@ -3,22 +3,32 @@
 !syntax description /Modules/TensorMechanics/Master/TensorMechanicsAction
 
 The TensorMechanics Master Action is a convenience object that simplifies part of the
-mechanics system setup. It performs
+mechanics system setup.
 
-- Add StressDivergence Kernels (for the current coordinate system)
-- Add WeakPlaneStress Kernel (for weak enforcement of the plane stress condition)
-- Add Strain calculation material (for the chosen strain model)
-- Correctly set use of displaced mesh
-- Optional: Setup of displacement variables (with the correct order for the current mesh)
-- Optional: Add AuxVariables and AuxKernels for various tensor components and quantity outputs
-- Optional: Set up out-of-plane stress/strain consistently
-- Optional: Automatic extraction of eigenstrain names from materials and correct application to proper blocks
+It applies to both the old kernel system based on the [StressDivergenceTensors](/StressDivergenceTensors.md) kernels
+and to the new kernel system based on the
+[TotalLagrangianStressDivergence](/TotalLagrangianStressDivergence.md) and [UpdatedLagrangianStressDivergence](/UpdatedLagrangianStressDivergence.md) kernels.
+Some options only apply to one or the other system, as outlined below.
+
+It performs
+
+- Add StressDivergence Kernels (for the current coordinate system) -- both systems, only Cartesian coordinates for the *Lagrangian* kernel system
+- Add WeakPlaneStress Kernel (for weak enforcement of the plane stress condition) -- only the `StressDivergenceTensors` system
+- Add Strain calculation material (for the chosen strain model) -- both systems
+- Correctly set use of displaced mesh -- both systems
+- Optional: Setup of displacement variables (with the correct order for the current mesh) -- both systems
+- Optional: Add AuxVariables and AuxKernels for various tensor components and quantity outputs -- both systems
+- Optional: Set up out-of-plane stress/strain consistently -- only the `StressDivergenceTensors` system
+- Optional: Automatic extraction of eigenstrain names from materials and correct application to proper blocks -- both systems
+- Optional: Setup cell-average homogenization constraints on the simulation -- only the new *Lagrangian* kernels
 
 ## Constructed MooseObjects
 
 The Tensor Mechanics `Master` Action is used to construct the kernels, displacement variables, and strain materials in a consistent manner as required for a continuum mechanics simulation simulation. Optionally it generates aux variables and auxkernels to aid in the output of tensor components and scalar quantities.
 
-!table id=tmMaster_action_table caption=Correspondence Among Action Functionality and MooseObjects for the Tensor Mechanics `Master` Action
+### For the `StressDivergenceTensors` Kernels
+
+!table id=tmMaster_action_table_sdt caption=Correspondence Among Action Functionality and MooseObjects for the Tensor Mechanics `Master` Action, old kernel system
```

Comment:
I'm hesitant to use the term 'old' to describe this since this is the only system that currently works for a lot of models, and it's not likely to go away any time soon. Maybe 'current'?

Reply:
"Current" it is.

Reply:
Sounds good. Once this all goes in, we'll still need to give more thought about the path forward with coexistence of the systems.